pkg/util/iptables/save_restore.go

```diff
@@ -52,7 +52,7 @@ func GetChainLines(table Table, save []byte) map[Chain]string {
 		} else if strings.HasPrefix(line, "#") {
 			continue
 		} else if strings.HasPrefix(line, ":") && len(line) > 1 {
-			chain := Chain(strings.SplitN(line[1:], " ", 2)[0])
+			chain := Chain(line[1:strings.Index(line, " ")])
```

Comment:
I think it is safe to assume, but maybe comment?  chain lines have 3 fields, space delimited.